examples/docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# You can also use your favorite Python Docker image.
+FROM python:3.8.4-slim
+ENV PYTHONIOENCODING UTF-8
```

Comment:
I think this is only needed for the Windows Server containers, and it's included upstream for those: https://github.com/docker-library/python/pull/557

Reply:
The Linux images get it from here: https://github.com/docker-library/python/blob/5590cdd4367f088277bb5494d0a0b0f65e9ab491/3.8/buster/Dockerfile#L14